react/Newsletter.tsx

```diff
@@ -94,6 +95,12 @@ function Newsletter(props: PropsWithChildren<Props>) {
       return
     }
 
+    triggerPixelNewsletterEvent({
```

Comment:
Use the hook usePixel, example:

1. https://github.com/vtex-apps/add-to-cart-button/blob/master/react/AddToCartButton.tsx#L12

2. https://github.com/vtex-apps/add-to-cart-button/blob/master/react/AddToCartButton.tsx#L120

3. https://github.com/vtex-apps/add-to-cart-button/blob/master/react/AddToCartButton.tsx#L192-L205

Reply:
@klzns Sure!
I will assume this thread from now on.

Btw I am also waiting for another PR to be accepted to move on with this feature 
https://github.com/vtex-apps/pixel-app-template/pull/15

Reply:
@klznsDone with the native hook as suggested, can you please review this part before I update the docs and remove the old implemetation files?

Reply:
Remember to remove old functions